packages/shared/src/vault/schemas.ts

```diff
@@ -14,6 +14,7 @@ const bytesToHex = (arr: Uint8Array | number[]) =>
 
 const utf8ToHex = (str: string) => `0x${Buffer.from(str).toString('hex')}`;
 
+// TODO: replace zod base validation with more readable custom validation
```

Comment:
we generate a zod schema here to validate all possible combinations for ccm params. this is not very readable and forces the consumer to cast (https://github.com/chainflip-io/chainflip-frontend-monorepo/blob/95bcb524a9352c2e87a47f0e17084e9288808615/packages/swap/integrations/chainflip.ts#L653-L656)

Reply:
for now it should be okay to just add usdt to the exising schema. but the schema will grow a lot when we add the arbitrum assets, so i would suggest refactoring this before we work on arbitrum